docs/jwst/pipeline/calwebb_spec2.rst

```diff
@@ -243,8 +243,13 @@ The input to the ``Spec2Pipeline`` pipeline is a countrate exposure, in the form
 of either "_rate" or "_rateints" data. A single input file can be processed or an
 ASN file listing multiple inputs can be used, in which case the processing steps
 will be applied to each input exposure, one at a time.
-If "_rateints" products are used as input, each step applies its algorithm to each
-integration in the exposure, where appropriate.
+If "_rateints" products are used as input, for modes other than NIRSpec Fixed Slit,
+each step applies its algorithm to each integration in the exposure, where appropriate.
+For the NIRSpec Fixed Slit mode, although the :ref:`flat_field <flat_field_step>`
+will accept "_rateints" products as input, the ``calwebb_spec2`` pipeline will currently
+fail at the :ref:`resample_spec <resample_step>` step which does not support
+multiple integration input products.  In the future, the :ref:`resample_spec <resample_step>`
+step will be modified to be skipped if its input is a "_rateints" product. 
```

Comment:
The future is now. We need a clean run through calwebb_spec2, not piecewise changes.

Reply:
Agreed. We can't merge code that we know will fail and describe the expected failure in the docs. Something has to be done to the code to avoid the failure, even if that just means adding logic to skip the resample_spec step.

Reply:
I agree with you both. I'm working on that code change right now.